modules/contentbox/modules/contentbox-admin/handlers/comments.cfc

```diff
@@ -196,9 +196,8 @@ component extends="baseHandler"{
 
 		// No comments selected
 		if( arrayLen( rc.commentID ) eq 0 ){
-			arrayAppend( data.message, "No comments selected!" );
 			data.error = true;
-			cbMessagebox.warn( data.messages );
+			cbMessagebox.warn( "No comments selected!" );
```

Comment:
You still need the `data.messages` to be set because if rendered by ajax, then the messagebox is not used.